src/queries/genericItem.js

```diff
@@ -187,9 +188,74 @@ export const GET_GENERIC_ITEM = gql`
       dates {
         id
         dateEnd
+        dateStart
       }
       publicationDate
       payload
     }
   }
 `;
+
+export const CREATE_GENERIC_ITEM = gql`
+  mutation createGenericItem(
+    $categoryName: String
+    $contacts: [ContactInput!]
+    $contentBlocks: [ContentBlockInput!]
+    $dates: [DateInput!]
+    $genericType: String
+    $publishedAt: String
+    $title: String
+  ) {
+    createGenericItem(
+      categoryName: $categoryName
+      contacts: $contacts
+      contentBlocks: $contentBlocks
+      dates: $dates
+      genericType: $genericType
+      publishedAt: $publishedAt
+      title: $title
+    ) {
+      contacts {
+        firstName
+        email
+      }
+      genericType
+      id
+      title
+      categories {
+        id
+        name
+      }
+      contentBlocks {
+        body
+        title
+      }
+      dates {
+        dateEnd
+        dateStart
+      }
```

Comment:
we do not need all the values returned, or am i wrong? please return only the values that we need on further processing or rendering

Reply:
Yes you are right. I only use them to check the correctness of the incoming data after creating a `genericItem`.

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/700/commits/7ff3cbd7b752a1cb9bcc1fc561a45887bd245ecc